### DISCUSSION TITLE
custom fields text name

### DISCUSSION THREAD

**Discussion by uRhaineWork** [netbox-community/netbox#9413] [Q&A] [answered]
Hi, in custom fields name field, can I somehow change letters to big or small where I want? Eg. now if I create custom field named 'DHCP' i get "Dhcp", which doesn't look good, could better. :) Or "Alternative_IP" -> I see "Alternative_ip".

**Comment by candlerb** [comment, marked as answer]
Yes: set the "Label" to whatever you want it to appear as.

![image](https://user-images.githubusercontent.com/44789/170072290-b66e69ec-224c-4785-8619-1a5338f09e89.png)
